src/analytics/patch/usePatchAnalytics.ts

```diff
@@ -11,6 +11,14 @@ import { GET_PATCH } from "gql/queries";
 
 type Action =
   | { name: "Filter Tasks"; filterBy: string }
+  | {
+      name: "Sort Tasks Table";
+      sortBy: "name" | "status" | "baseStatus" | "variant";
```

Comment:
Can we use these values from https://github.com/evergreen-ci/spruce/blob/4c6089b3310c9103bbbee182e035a6249e652aa3/src/gql/generated/types.ts#L525-L530